cfme/configure/tasks.py

```diff
@@ -160,3 +160,47 @@ def all_other_tasks(**filter_kwargs):
     Returns: List of dicts.
     """
     return _get_tasks("tasks_all_other", **filter_kwargs)
+
```

Comment:
Yep, we can make it generic, however I found no other tests with task watching. I guess we can make it generic for other tasks too